docs/sources/explore/trace-integration.md

```diff
@@ -115,7 +115,7 @@ Click the document icon to open a split view in Explore with the configured data
 This feature is currently in beta & behind the `traceToMetrics` feature toggle.
 {{% /admonition %}}
 
-You can navigate from a span in a trace view directly to metrics relevant for that span. This feature is available for Tempo, Jaeger, and Zipkin data sources. Refer to their [relevant documentation](/docs/grafana/latest/datasources/tempo/#trace-to-metrics) for configuration instructions.
+You can navigate from a span in a trace view directly to metrics relevant for that span. This feature is available for Tempo, Jaeger, and Zipkin data sources. Refer to their [relevant documentation](https://grafana.com/docs/grafana/latest/datasources/tempo/configure-tempo-data-source/#trace-to-metrics) for configuration instructions.
```

Comment:
there are relative URLs all over the place in this file, I feel like you should have this one be relative too

Reply:
Sure I will be happy to make this a relative link, I just need to know how to do that.  Sorry, I am not familiar with this process :)

Reply:
Ah, there's no process. Look at the diff - what was there before was a relative link, and by adding the protocol and domain, it's an absolute link. You should just have to remove `https://grafana.com` from the URL and be good to go

Reply:
Cool, thanks for letting me know.  I have made this change.